pkg/server/gen/server.swagger.json

```diff
@@ -6204,6 +6267,122 @@
         ]
       }
     },
+    "/ui/project/{project.project}/pipelines": {
```

Comment:
Not strictly related to this change. Looks like we forgot to do a final `make gen/server` as part of #4509.

Reply:
I'm a bit unclear on what happened. We did generate a bunch of stuff, so when was this step supposed to happen?